components/member-profile/contribution/index.js

```diff
@@ -42,7 +42,11 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
   }
 
   return (
-    <div>
+    <div
+      className={classNames.contributionCard}
+      onClick={() => window.open(featureUrl || prList[0]['url'], '_blank')}
```

Comment:
I can see the logic for `featureUrl || prList[0]['url']` being repeated in two places
Can we move them to a common variable/function?

Also, what happens when we have no featureUrl and no prList elements? Do we have a runtime error?